vision/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -84,29 +85,29 @@ public void testFacesGcs() throws Exception {
   @Test
   public void testLabels() throws Exception {
     // Act
-    String[] args = {"labels", "./resources/wakeupcat.jpg"};
+    String[] args = { "labels", "./resources/wakeupcat.jpg" };
     Detect.argsHelper(args, out);
 
     // Assert
     String got = bout.toString();
-    assertThat(got).contains("whiskers");
+    assertThat(got).contains("Whiskers");
   }
 
   @Test
   public void testLabelsGcs() throws Exception {
     // Act
-    String[] args = {"labels", "gs://" + BUCKET + "/vision/wakeupcat.jpg"};
+    String[] args = { "labels", "gs://" + ASSET_BUCKET + "/vision/label/wakeupcat.jpg" };
     Detect.argsHelper(args, out);
 
     // Assert
     String got = bout.toString();
-    assertThat(got).contains("whiskers");
+    assertThat(got).contains("Whiskers");
```

Comment:
If this (and others below) is going to be case sensitive, might we want to just lowercase the `got` and `expected` values for the comparison?

Reply:
Will relax further.